deepchem/models/torch_models/modular.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, model: nn.Module, components: dict, **kwargs):
         super().__init__(self.model, self.loss_func, **kwargs)
         self.model.to(self.device)
         self.components = {
-            k: v.to(self.device) for k, v in self.components.items()
+            k: v.to(self.device) for k, v in self.components.items() if isinstance(v, nn.Module)
```

Comment:
When would we expect to have a non `nn.module` in self.components?

Reply:
In this GNNModular class. The pooling components can be either a function or parameterized nn.module.